packages/prep_pkg.R

```diff
@@ -138,7 +138,7 @@ cat(paste(text, collapse = "\n"), file = file.path("nimble", "man", "nimble-math
 # now create real NAMESPACE file
 ## NAMESPACE
 system(paste("mv", file.path('nimble','NAMESPACE'), file.path('/tmp', 'NAMESPACE')))
-roxygenise('nimble','namespace')
+devtools::document('nimble', roclets = c('namespace'))
```

Comment:
I'm merging this but am then going to do some revamping of prep_pkg.R to more cleanly use roxygen2. 

Note that we'll need to use roxygen2 v5.0.1 because of issues with v6.0.1 noted earlier in this thread (I also had problems with devtools::document, though we could revisit that and see if that would work instead of falling back to older roxygen2...)